catalogs/catalog-lakehouse-paimon/src/main/java/com/datastrato/gravitino/catalog/lakehouse/paimon/GravitinoPaimonColumn.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.lakehouse.paimon;
+
+import static com.datastrato.gravitino.catalog.lakehouse.paimon.utils.TypeUtils.fromPaimonType;
+import static com.datastrato.gravitino.catalog.lakehouse.paimon.utils.TypeUtils.toPaimonType;
+
+import com.datastrato.gravitino.connector.BaseColumn;
+import com.datastrato.gravitino.rel.Column;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.EqualsAndHashCode;
+import org.apache.paimon.types.DataField;
+import org.apache.paimon.types.DataType;
+import org.apache.paimon.types.RowType;
+
+/** Implementation of {@link Column} that represents a column in the Paimon column. */
+@EqualsAndHashCode(callSuper = true)
+public class GravitinoPaimonColumn extends BaseColumn {
+
+  private GravitinoPaimonColumn() {}
+
+  /**
+   * Converts {@link GravitinoPaimonColumn} instance to inner column.
```

Comment:
The meaning of `inner column` is not clear, please polish the java doc.

Reply:
updated.